VOM_Fortran/VOM-code/transpmodel.f90

```diff
@@ -1097,38 +1102,50 @@ subroutine vom_daily_init ()
       jmax25g_d(:) = jmax25g_d(2) * (/1.0d0-i_incrjmax,1.0d0,1.0d0+i_incrjmax/)
       jmax25g_d(:) = MAX(jmax25g_d(:), 50.0d-6)
 
+!     * adjust lai-values trees
+      lai_lt(:) = lai_lt(2) * (/1.0d0-i_incrlait,1.0d0,1.0d0+i_incrlait/)
+      lai_lt(:) = MAX( lai_lt(:), 0.1d0 ) !minimum value, else lai doesn't pick up anymore
```

Comment:
You might want to set the minimum to a smaller value, perhaps even supplied in the input file, along with i_incrlait.

Reply:
Yes, good point, will do that.